ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -129,7 +129,11 @@
     /**
      * User metadata on its own i.e., no "blob properties" headers returned with response.
      */
-    UserMetadata
+    UserMetadata,
+    /**
+     * All the replicas of the blob ID returned as content (Admin only).
+     */
+    Replicas
```

Comment:
I would suggest add more documentation on this subresource. Initially it was confusing to me, as I thought different meaning for it actually is. Now I understand this is to get the string representation of all replicas of a blob id, instead of the actual replicas of a blob id (and I thought why need to do so...). So, the call with replicas as subresource is simply a call without the help of router, and just get the metadata information of all replicas.